pkg/maintainer/wallet/chain_test.go

```diff
@@ -16,72 +19,290 @@ type walletLock struct {
 	walletAction   tbtc.WalletActionType
 }
 
-type localChain struct {
+type depositParameters = struct {
+	dustThreshold      uint64
+	treasuryFeeDivisor uint64
+	txMaxFee           uint64
+	revealAheadPeriod  uint32
+}
+
+type LocalChain struct {
```

Comment:
Some tests in `pkg/maintainer/wallet` lives in the `wallet_test` package. This is necessary to avoid import cycles with `pkg/maintainer/wallet/internal/test`. Hence the need of exporting `LocalChain`.

Reply:
Would it help to move the test chain implementation to `internal/local`? 🤔

Reply:
I don't think so as we would still have to export `LocalChain` from there.

Reply:
An ultimate solution would be to rebuild `pkg/maintainer/wallet/internal/test` to not depend on `pkg/maintainer/wallet` types but not sure how much work is needed to do so.